public/manifest.json

```diff
@@ -12,4 +12,4 @@
   "display": "standalone",
   "theme_color": "#000000",
   "background_color": "#ffffff"
-}
+}
```

Comment:
Check files added to your commit, and remove ones that are not changed. If you add all files like these which do not really have changes also get added. Try using the terminal

Reply:
yup, fixed this now on the new PR